library/modelfiles/nous-hermes

```diff
@@ -3,5 +3,12 @@ TEMPLATE """
 ### Instruction:
 {{ .Prompt }}
 
+{{- if and .First .System }}
+### Input:
+{{ .System }}
+{{- end }}
+
 ### Response:
 """
+
+SYSTEM """You are an AI assistant that follows instruction extremely well. Help as much as you can."""
```

Comment:
There shouldn't be a default SYSTEM. I suggest this be removed

Reply:
Why is nous-hermes different than all the others? llam2, orca, vicuna have a default SYSTEM

Reply:
I think this shows that `SYSTEM` might not be the best wording for this in general. I think it should go in the `Instruction` part of this template:

```
### Instruction:
{{ .System }}
### Input:
{{ .Prompt }}
### Response:
```

source: https://huggingface.co/NousResearch/Nous-Hermes-13b

Reply:
@BruceMacD you're right, my mistake